content/tracing/languages/java.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Tracing Java Applications
+kind: Documentation
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-java"
+  tag: "Github"
+  text: Source code
+- link: "/tracing/"
+  tag: "Documentation"
+  text: APM (Tracing)
+---
+
+## Compatibility
+
+We officially support Java 1.7 and higher of both Oracle JDK and OpenJDK.  To see what web frameworks, libraries, and datastores we support, see the Integrations section below.
```

Comment:
Can we link to the Integrations section where it says "see the Integrations section below?"

Reply:
👍